frontend/src/components/Home.js

```diff
@@ -62,7 +62,20 @@ export default class Home extends Component {
             window.location.replace('http://localhost:8000/')
         }, 1000);
     }
-
```

Comment:
added the getCurrentSong() function to see if it would help with retriving the artist info but currently is not working